docs/contributing/testing.md

```diff
@@ -0,0 +1,112 @@
+# Test your GOV.UK Frontend contribution
+
+Before you submit your contribution, you should:
+
+- check your changes look right and behave correctly
+- test in supported browsers and assistive technology
+- run our automated tests, and update tests or add new tests if our tests fail
```

Comment:
A note on language here, could we say something like `run the automated tests` rather than `our`, it feels a bit distant when we want a contributor to feel like they're doing work alongside us? If they're contributing it's their tests too? :angel: